audit-service/src/contract-tests/java/uk/gov/gchq/palisade/contract/audit/kafka/KafkaContractTest.java

```diff
@@ -104,33 +114,30 @@
 @Import(KafkaContractTest.KafkaInitializer.Config.class)
 @ContextConfiguration(initializers = {KafkaContractTest.KafkaInitializer.class})
 @ActiveProfiles({"akka-test"})
+@ExtendWith(MockitoExtension.class)
 class KafkaContractTest {
 
     private static final Logger LOGGER = LoggerFactory.getLogger(KafkaContractTest.class);
-    private static final ObjectMapper MAPPER = new ObjectMapper();
 
     // Serialiser for upstream test input
     static class RequestSerializer implements Serializer<JsonNode> {
+        private final ObjectMapper objectMapper = new ObjectMapper();
```

Comment:
Why are we not using the ObjectMapper that is already wired in with Spring?

Reply:
I'll try and inject it.

Reply:
Took a bit of trial and error, but came up with adding two factory methods to the Config class for the serialisers. Now the ObjectMapper is injected into the value serialiser from the Spring context. Also removed the need for the static nested `RequestSerialiser` class. Much better.